include/bio_ik/goal_types.h

```diff
@@ -213,95 +213,95 @@ class LookAtGoal : public LinkGoalBase
 
 class MaxDistanceGoal : public LinkGoalBase
 {
-    tf2::Vector3 target;
-    double distance;
+    tf2::Vector3 target_;
+    double distance_;
```

Comment:
Do we want to include these sorts of stylistic changes in this PR? I'm not sure to what extent we want to impose PickNik style on code that has not yet begin given to us.